doc/Media2.xml

```diff
@@ -2291,7 +2529,22 @@
           </listitem>
         </varlistentry>
       </variablelist>
-    </section>
+    <para>The following audio clip Capabilities are available:</para>
```

Comment:
All these capabilities are attributes in the schema, so they are not mandatory by default. What it will mean if a DUT does not send MaxAudioClipLimit (no limit?), MaxAudioClipSize (no limit?), SupportedAudioClipFormat (all from SupportedAudioClipFormat are supported?)

Reply:
AudioClipCapabilities  area made as optional to main backward capability. Once AudioClipCapabilities  are listed, the parameters in  AudioClipCapabilities   i.e MaxAudioClipLimit , MaxAudioClipSize , SupportedAudioClipFormat  are mandatory from the device. It is similar to AuthorizationServer  capabilities added in ONVIF advance security service.